Automattic/pocket-casts-android
# Integrate `measure-builds` plugin

## Description


This PR integrates [`measure-builds-gradle-plugin`](https://github.com/Automattic/measure-builds-gradle-plugin) to collect build metrics.

## Testing Instructions




1. Make sure you updated secrets (`bundle exec fastlane run configure_apply`), and you have `appsMetricsToken` value in `secret.properties`.
2. Run any task with `--debug` flag, e.g. `./gradlew help --debug`
3. Find `REQUEST: https://metrics.a8c-ci.services/api/grouped-metrics` log

Please mind that the connection **can time out** due to a backend issue. Please don't worry about it, it's a known issue that we'll address it later.

## Screenshots or Screencast 

 <img width="590" alt="Screenshot 2023-12-11 at 12 47 48" src="https://github.com/Automattic/pocket-casts-android/assets/5845095/6a7660e7-6515-4eca-8723-4726604cd14e">
